src/features/views/components/ViewDataTable/columnTypes/LocalPersonColumnType.tsx

```diff
@@ -57,7 +57,15 @@ export default class LocalPersonColumnType
         return makeName(v1).localeCompare(makeName(v2));
       },
       valueGetter: (params) => {
-        return params.value ? makeName(params.value) : '';
+        // we add a `toString`-method, which will be used when filtering for LocalPersons
+        return params.value
+          ? {
+              ...params.value,
+              toString() {
```

Comment:
This is such a smart solution! How did you figure out to do it this way? It's so neat!